### DISCUSSION TITLE
Auto Pilot

### DISCUSSION THREAD

**Discussion by henkscheffer** [flybywiresim/aircraft#5173] [Help]
### Aircraft Version

Development

### Build info

```json
"built": "2021-06-10T09:49:19+00:00",
  "ref": "master",
  "sha": "1578f4fa852800ef1d7c46ad6460f21bf92a2c3a",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

Since a week, when I use the Auto Pilot, after a few moments or after some time, the Auto Pilot disappears. Than I have real trouble getting the plane under control again and set the Auto Pilot.  What is wrong? How can you help.

### Expected behavior

Some advice to solve the poroblem

### Steps to reproduce

As I say, suddenly, without warning the AP disappears. 

### References (optional)

I use the development version

### Discord Username (optional)

WAXIESTCOYOTE53

**Comment by derl30n** [comment]
Can you please provide the corresponding FDR file from `\AppData\Roaming\Microsoft Flight Simulator\Packages\flybywire-aircraft-a320-neo\work`?

**Comment by henkscheffer** [comment]
Good morning,

I have 16 corresponding FDR files. E.g one of these files from Juni 11th is
bigger than 25mb. I will send you the two smallest. If you need more, I
will send them by WeTransfer.
Have a nice weekend

Henk Scheffer

Op vr 11 jun. 2021 om 19:11 schreef DerL30N ***@***.***>:

**Comment by aguther** [comment]
Please send the bigger ones, the small ones are pretty sure useless because the sim often reloads the plane without the user seeing it and that triggers creation of a new file (during loading a flight or going back to main menu).

Maybe also a more detailed description and screenshots or video of what’s happening would be helpful in the first place…

**Comment by henkscheffer** [comment]
Goed afternoon,

I wonder if you have received  the FDR files I send you by WeTransfer?.

Greetings

Henk Scheffer



Verstuurd vanaf mijn iPhone

**Comment by aguther** [comment]
I did not receive anything…did you provide a link?

**Comment by henkscheffer** [comment]
Sorry to hear that. I send it by WeTransfer, because the files were to big. Is there another possibility? If I send it bij email, I have a maximum of 25 mb.

Henk Scheffer


Verstuurd vanaf mijn iPhone

**Comment by aguther** [comment]
https://www.swisstransfer.com/en

**Comment by henkscheffer** [comment]
Good morning,

I only want to know if you have now received the files. I have send them by Swisstransfer, as you asked.

Henk Scheffer

Verstuurd vanaf mijn iPhone

**Comment by aguther** [comment]
You may have uploaded them, but without the link to the files there is no way to download them. Just post the link here, the files contain no personal information.

**Comment by henkscheffer** [comment]
Sorry, 
I did not understand what you mean with “post the link here”. (It’s my lack of this kind of English). Which link do you mean. Can you give me an example?

Henk Scheffer



Verstuurd vanaf mijn iPhone

**Comment by CoffeePanda0** [comment]
> 
> Sorry, I did not understand what you mean with “post the link here”. (It’s my lack of this kind of English). Which link do you mean. Can you give me an example? Henk Scheffer Verstuurd vanaf mijn iPhone
> […](#)
> Op 16 jun. 2021 om 12:22 heeft Andreas Guther ***@***.***> het volgende geschreven: ﻿ You may have uploaded them, but with the link to the files there is no way to download them. Just post the link here, the files contain no personal information. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

He means to post the link to the swisstransfer files that you uploaded as a comment here on this github question

**Comment by henkscheffer** [comment]
I hope this is what you mean

C:\Users\henk-\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\flybywire-aircraft-a320-neo\work



Op wo 16 jun. 2021 om 14:26 schreef Alexander Thomas <
***@***.***>:

**Comment by aguther** [comment]
No, that's your local path how should I access this one?

If you uploaded the files to SwissTransfer when it's finished you get a link you can share so others can download it. That's the one we need.
